src/seedu/addressbook/data/person/Name.java

```diff
@@ -64,7 +64,7 @@ public int hashCode() {
      * Returns true of the other name is very similar to this name.
      * Two names are considered similar if ...
      */
-     public boolean isSimilar(Name other) {
+     public static boolean isSimilar(Name other) {
```

Comment:
if you realised what you wanted (which you write in the test) in the first place, this wouldn't happen (: #TDD